epl-nft/contracts/EnglishPremierLeague.cdc

```diff
@@ -802,6 +800,12 @@ pub contract EnglishPremierLeague: NonFungibleToken {
             return <- momentNFT
         }
 
+        /// Purchase Pack
+        ///
+        pub fun purchasePack(purchaseAddress: Address, packID: UInt64) {
```

Comment:
is there a way to make this method more secured?

Reply:
maybe make it so that only the account the contract is deployed to can call it, then the purchase tx will be a multisig tx